inspirehep/modules/theme/views.py

```diff
@@ -101,3 +116,40 @@ def data():
 @blueprint.route('/ping')
 def ping():
     return 'OK'
+
+
+#
+# Feedback
+#
+
+@blueprint.route('/postfeedback', methods=['POST', ])
+def postfeedback():
+    """Handler to create a ticket from user feedback."""
+    feedback = request.form.get('feedback')
+    if feedback == '':
+        return jsonify(success=False), 400
+
+    replytoaddr = request.form.get('replytoaddr', None)
+    if replytoaddr is None:
+        if current_user.is_anonymous:
+            return jsonify(success=False), 403
+        else:
+            replytoaddr = current_user.get('email')
+            if replytoaddr == '':
+                return jsonify(success=False), 403
+
+    content = 'Feedback:\n{feedback}'.format(feedback=feedback)
+    message = {
+        'sender': current_app.config['CFG_SITE_SUPPORT_EMAIL'],
+        'recipients': [current_app.config['INSPIRELABS_FEEDBACK_EMAIL']],
+        'subject': 'INSPIRE Labs Feedback',
+        'body': content,
+        'reply_to': replytoaddr
+    }
+
+    sending = send_email.delay(message)
+
+    if sending.failed():
```

Comment:
In this case, yes, since `send_email` just tries to send an email and returns. If the task had more logic then we could verify some properties of the email being sent, validate them and set the `state` of the task — but the more work we do, the less async the task is, as you say.